src/transaction/log_page_buffer.c

```diff
@@ -3541,7 +3541,7 @@ logpb_catchup_finish (THREAD_ENTRY * thread_p, const LOG_LSA catchup_lsa)
   assert (LOG_CS_OWN_WRITE_MODE (thread_p));
   assert (log_Pb.partial_append.status = LOGPB_APPENDREC_SUCCESS);
 
-  char log_pgbuf[LOG_PAGESIZE + MAX_ALIGNMENT], *aligned_log_pgbuf;
+  char log_pgbuf[IO_MAX_PAGE_SIZE + MAX_ALIGNMENT], *aligned_log_pgbuf;
```

Comment:
Why does this change come along?

Reply:
@wrlawodms 
It fixes the Visual C++ build.
See the build logs of [8350806](https://github.com/CUBRID/cubrid/pull/4764/commits/8350806d4824fbcd438d7498dfb3c709d77a13a6) above - see https://app.circleci.com/pipelines/github/CUBRID/cubrid/11650/workflows/8e3446dc-88fe-40d8-9cc7-dd4d7cc130b8/jobs/61797

```
C:\Users\circleci\project\src\transaction\log_page_buffer.c(3544,31): error C2131: expression did not evaluate to a constant [C:\Users\circleci\project\build\cubrid\cubrid.vcxproj]
C:\Users\circleci\project\src\transaction\log_page_buffer.c(3544,18): message : failure was caused by non-constant arguments or reference to a non-constant symbol [C:\Users\circleci\project\build\cubrid\cubrid.vcxproj]
```

`LOG_PAGESIZE` is not a compile time constant:
```
#define LOG_PAGESIZE            db_Log_page_size
```

I just replaced it with `IO_MAX_PAGE_SIZE` as is done in other functions in `log_page_buffer`.
I am surprised as why g++ does not treat this as an error.

References:
- https://stackoverflow.com/questions/33423502/expression-did-not-evaluate-to-a-constant-c
- https://stackoverflow.com/questions/63558400/vs2019-latest-update-constexpr-expression-did-not-evaluate-to-a-constant-for

Reply:
Oh, it's not allowed in standard. I wondered why `IO_MAX_PAGE_SIZE` is used everywhere. Now it's solved.